File: RepoTransBench/target_projects/Java/Go/apache_rocketmq-spring/tests/original/ext_rocketmq_template_test.go
package original

import (
	"testing"

	"github.com/stretchr/testify/assert"
)

type ExtRocketMQTemplate struct{}

func TestExtRocketMQTemplateInstantiation(t *testing.T) {
	template := &ExtRocketMQTemplate{}
	assert.NotNil(t, template)
}

File: RepoTransBench/source_projects/Java/apache_rocketmq-spring/rocketmq-spring-boot-samples/rocketmq-consume-demo/src/test/java/org/apache/rocketmq/samples/springboot/ExtRocketMQTemplateTest.java
package org.apache.rocketmq.samples.springboot;

import org.junit.jupiter.api.Test;

public class ExtRocketMQTemplateTest {

    @Test
    public void testExtRocketMQTemplateInstantiation() {
        ExtRocketMQTemplate template = new ExtRocketMQTemplate();
        // Just instantiation test, as ExtRocketMQTemplate has no complex logic
        assert template != null;
    }
}